packages/flutter/lib/src/animation/forces.dart

```diff
@@ -10,6 +10,7 @@ export 'package:newton/newton.dart' show SpringDescription;
 abstract class Force {
   const Force();
 
+  /// Creates a new physics simulations with the given initial conditions.
```

Comment:
"physics simulation" is really vague. What kind of simulation? Spring? Newtonian acceleration?